qa/tasks/cephfs/kernel_mount.py

```diff
@@ -210,10 +210,10 @@ def get_id_to_dir():
             print(json.dumps(get_id_to_dir()))
             """)
 
-        p = self.client_remote.run(args=[
-            'sudo', 'python', '-c', pyscript
-        ], stdout=StringIO(), timeout=(5*60))
-        client_id_to_dir = json.loads(p.stdout.getvalue())
+        output = self.client_remote.sh([
```

Comment:
this change should have two backport of follow commits

- 3c751fea7f48ef613286211797a7eafc4913b36e
- 875b45f3c903781ba99dbd2fcf4d2fc89001c25b

the root cause is

- i forgot to backport 3c751fea7f48ef613286211797a7eafc4913b36e
- i failed to correctly backport 875b45f3c903781ba99dbd2fcf4d2fc89001c25b to nautilus correctly

Reply:
Thanks a lot, Kefu!

Reply:
please either backport 875b45f3c903781ba99dbd2fcf4d2fc89001c25b or note down the reason why this change is not cherry-picked from master in the commit message.

Reply:
Mentioned in the commit message that it was an incomplete backport and that it missed some commit line.